assemblies/server/src/main/resources-filtered/etc/org.apache.karaf.features.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<featuresProcessing xmlns="http://karaf.apache.org/xmlns/features-processing/v1.0.0" xmlns:f="http://karaf.apache.org/xmlns/features/v1.6.0">
+
```

Comment:
Add a comment referring the common `org.apache.karaf.features.xml` and that it should be used as base and "kept in sync".